libconnman-qt/networkservice.h

```diff
@@ -12,6 +12,7 @@
 #define NETWORKSERVICE_H
 
 #include <QObject>
+#include <QDBusObjectPath>
```

Comment:
Shoud this go to the cpp file now because the moveBefore and moveAfter don't have QDBusObjectPath as parameter type?

Reply:
Make sense. But I was able to compile without any warnings by dropping it from the header file (and not adding it to the .cpp file). Should we drop it completely?

Reply:
Seems to be ok to remove, because that is included in commondbustypes.h which gets included via libconnman_p.h

Reply:
Great, issue corrected!